FirebaseAdmin/FirebaseAdmin.Tests/Auth/Providers/SamlProviderConfigTest.cs

```diff
@@ -404,5 +526,119 @@ public IEnumerator<object[]> GetEnumerator()
+                // {
+                //    1st element: InvalidInput,
+                //    2nd element: ExpectedError,
+                // }
+                yield return new object[]
+                {
+                    new SamlProviderConfigArgs(),
+                    "Provider ID cannot be null or empty.",
+                };
+                yield return new object[]
+                {
+                    new SamlProviderConfigArgs()
+                    {
+                        ProviderId = string.Empty,
+                    },
+                    "Provider ID cannot be null or empty.",
+                };
+                yield return new object[]
+                {
+                    new SamlProviderConfigArgs()
+                    {
+                        ProviderId = "oidc.provider",
+                    },
+                    "SAML provider ID must have the prefix 'saml.'.",
+                };
+                yield return new object[]
+                {
+                    new SamlProviderConfigArgs()
+                    {
+                        ProviderId = "saml.provider",
+                    },
+                    "At least one field must be specified for update.",
+                };
+                yield return new object[]
+                {
+                    new SamlProviderConfigArgs()
+                    {
+                        ProviderId = "saml.provider",
+                        IdpEntityId = string.Empty,
+                    },
+                    "IDP entity ID must not be empty.",
+                };
+                yield return new object[]
+                {
+                    new SamlProviderConfigArgs()
+                    {
+                        ProviderId = "saml.provider",
+                        SsoUrl = string.Empty,
+                    },
+                    "SSO URL must not be empty.",
+                };
+                yield return new object[]
+                {
+                    new SamlProviderConfigArgs()
+                    {
+                        ProviderId = "saml.provider",
+                        SsoUrl = "not a url",
+                    },
+                    "Malformed SSO URL: not a url",
+                };
+                yield return new object[]
+                {
+                    new SamlProviderConfigArgs()
+                    {
+                        ProviderId = "saml.provider",
+                        X509Certificates = new List<string>(),
+                    },
+                    "X509 certificates must not be empty.",
+                };
+                yield return new object[]
+                {
+                    new SamlProviderConfigArgs()
+                    {
+                        ProviderId = "saml.provider",
+                        X509Certificates = new List<string>() { string.Empty },
+                    },
+                    "X509 certificates must not contain null or empty values.",
```

Comment:
Is there a unit test for the null value check?

Reply:
Done